app/lib/search/index_simple.dart

```diff
@@ -282,7 +285,13 @@ class TokenIndex {
     }
     for (String id in counts.keys.toList()) {
       final double current = counts[id];
-      counts[id] = 100.0 * (current / _weights[id]) * (current / sumWeight);
+      double score = 100.0 * (current / sumWeight);
+      if (idfWeight > 0.0) {
+        // adjust score with inverse document frequency
+        final double idf = current / _weights[id];
```

Comment:
The term-frequency `tf(term, document)` for each term and document can also be pre-computed when building the index.

Reply:
So, here we are a bit sloppy, and this is not really true TF-IDF. Making it a true TF-IDF was on my roadmap, but it never was a priority, and worked out okay-ish so far, and the two additional steps for it doesn't seemed to be justified (neither in complexity, nor time involved). I'm making it more obvious in the documentation, PTAL, and let me know if you'd prefer true TF-IDF (I'd guess it shall be after merging and deploying this).